views/dashboard/_keywords.html

```diff
@@ -5,14 +5,16 @@
         <th>Keyword</th>
         <th>Search Completed</th>
         <th>Created At</th>
+        <th>Action</th>
       </tr>
     </thead>
     <tbody>
       {{ range $keyword := .Keywords }}
       <tr>
-        <td>{{ $keyword.Name }}</td>
-        <td>{{ $keyword | displayStatus }}</td>
-        <td>{{ $keyword | displayFormattedCreatedDate }}</td>
+        <td class="align-middle">{{ $keyword.Name }}</td>
+        <td class="align-middle">{{ $keyword | displayStatus }}</td>
+        <td class="align-middle">{{ $keyword | displayFormattedCreatedDate }}</td>
+        <td><a href="/keyword/{{ $keyword.Id }}" class="btn btn-primary btn-sm">Show</a></td>
```

Comment:
@junan You can try with this, it really useful. ✌️
```html
{{ urlfor `KeywordController.Show` `:id` $keyword.Id }}
```

Reply:
I know it, but I prefer current one as it is simple and readable ✌️

Reply:
👍